types/mysql/index.d.ts

```diff
@@ -1,177 +1,268 @@
-// Type definitions for node-mysql
+// Type definitions for node-mysql 2.15.0
 // Project: https://github.com/felixge/node-mysql
 // Definitions by: William Johnston <https://github.com/wjohnsto>
 //		   Kacper Polak <https://github.com/kacepe>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-
+// TypeScript Version: 2.5
```

Comment:
Is this needed? Could you possibly use a lower version number (or none, which means ts2.0)?

Reply:
removed